Pod/Classes/NWSTokenView.swift

```diff
@@ -108,6 +126,14 @@ public class NWSTokenView: UIView, UIScrollViewDelegate, UITextViewDelegate
         NSNotificationCenter.defaultCenter().addObserver(self, selector: "didRotateInterfaceOrientation", name: UIDeviceOrientationDidChangeNotification, object: nil)
     }
     
+    public override func layoutSubviews() {
+        super.layoutSubviews()
+        
+        if self.tokens.count == 0 {
+            self.scrollView.contentSize = CGSizeMake(self.scrollView.bounds.width, self.scrollView.bounds.height)
```

Comment:
@wfilleman Whats the reasoning for this? Not against it, just don't see what its doing differently.